cpp/util/etcd_test.cc

```diff
@@ -184,53 +169,79 @@ class EtcdTest : public ::testing::Test {
   }
 
   const shared_ptr<libevent::Base> base_;
+  StrictMock<MockUrlFetcher> url_fetcher_;
   libevent::EventPumpThread pump_;
-  MockEtcdClient client_;
-  const map<string, string> kEmptyParams;
+  EtcdClient client_;
 };
 
-void GenericReturn(Status status, const shared_ptr<JsonObject>& json_body,
-                   int64_t etcd_index, EtcdClient::GenericResponse* resp,
-                   Task* task) {
-  resp->etcd_index = etcd_index;
-  resp->json_body = json_body;
+string GetEtcdUrl(const string& key) {
+  return "http://" + string(kEtcdHost) + ":" + to_string(kEtcdPort) +
```

Comment:
Worth a `CHECK(!key.empty() && key[0] == '/')`here just to avoid potential typo-induced head-scratchers?